src/main/java/org/employeeBook/Second/DepartmentController.java

```diff
@@ -0,0 +1,35 @@
+package org.employeeBook.Second;
+
+import org.employeeBook.Employee;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/departments")
+public class DepartmentController {
+    private final DepartmentServiceImpl departmentServiceImpl;
```

Comment:
```suggestion
    private final DepartmentService departmentServiceImpl;
```
инъекцию надо делать по интерфейсу, а не реализации